### PR TITLE
Re: #2160 Add UGC navbar to object page

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: app/views/portal/show.rb [hunk 1]
```diff
@@ -83,7 +83,11 @@ def content
           refs_rels: presenter.field_group(:refs_rels),
           similar: similar_items,
           named_entities: named_entities,
-          thumbnail: field_value('europeanaAggregation.edmPreview', tag: false)
+          thumbnail: field_value('europeanaAggregation.edmPreview', tag: false),
+          ugc_content: {
+            base_1418_url: 'http://ec2-54-154-38-184.eu-west-1.compute.amazonaws.com',
```
Comment (lbiedinger) [inline app/views/portal/show.rb]: instead of hardcoding here, use `config.x.europeana_1914_1918_url`

**Comment by andyjmaclean** [comment]
@rwd the recommended view relies on @collection which is always null in this context

**Comment by andyjmaclean** [comment]
Added a parameter to ugc_displaying view allowing it to be forced when no collection is present

File: app/views/concerns/ugc_content_displaying_view.rb [hunk 2]
```diff
@@ -7,8 +7,8 @@ module UgcContentDisplayingView
 
   protected
 
-  def ugc_content
-    if @collection && @collection.accepts_ugc?
+  def ugc_content(force)
```
Comment (lbiedinger) [inline app/views/concerns/ugc_content_displaying_view.rb]: `def ugc_content(force = false)`

**Pull request merged**